management/src/main/java/org/kaazing/gateway/management/monitoring/configuration/impl/DefaultMMFMonitoringDataManagerStub.java

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
This class is just an empty implementation of MonitoringDataManager and has no MMF awareness so it can just be called MonitoringDataManagerStub.